Utilities/Mergers/src/MergerBuilder.cxx

```diff
@@ -66,20 +69,20 @@ framework::DataProcessorSpec MergerBuilder::buildSpec()
   merger.inputs = mInputSpecs;
 
   merger.outputs.push_back(mOutputSpec);
-  if (mOutputSpec.origin == header::gDataOriginInvalid || mOutputSpec.description == header::gDataDescriptionInvalid) {
+  framework::DataAllocator::SubSpecificationType subSpec;
```

Comment:
@knopers8 if I do `subSpec = 0` you should be fine, no?

Reply:
Only if we assume that a Merger in the last layer was specified to publish on subspec equal 0, which might not be necessarily the case.

Reply:
Ok, I've added `= DataSpecUtils::getOptionalSubSpec(mOutputSpec).value();` we can add the protection for non specified subSpec in the other PR, where it would actually matter...